util/json.go

```diff
@@ -30,6 +31,19 @@ func RespondWithJSON(w http.ResponseWriter, code int, payload interface{}) {
 	w.Write(response)
 }
 
+func RespondRaw(w http.ResponseWriter, code int, payload []byte) {
```

Comment:
This should be shared with the various CLI tools. So that we only have to change it once and not multiple times.

Reply:
It's exported within the util package, so it should be able to be used anywhere that it's necessary.  What other tools should use it?